pkg/clusteragent/admission/common/const.go

```diff
@@ -38,4 +38,7 @@ const (
 
 	// LibConfigV1AnnotKeyFormat is the format of the library config annotation
 	LibConfigV1AnnotKeyFormat = "admission.datadoghq.com/%s-lib.config.v1"
+
+	// KubeSystemNamespaceLabelKey is the label to select the kube-system namespace.
+	KubeSystemNamespaceLabelKey = "kubernetes.io/metadata.name"
```

Comment:
According to the docs this was added in 1.22, so already a long time ago, but it might be worth adding a comment here or in the label selector function to remind us that this is not going to work if the user is on an old version of kubernetes: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/#automatic-labelling